terraform/concourse/elb.tf

```diff
@@ -1,3 +1,13 @@
+resource "aws_iam_server_certificate" "concourse" {
+  name = "${var.env}-concourse"
+  certificate_body = "${file("concourse.crt")}"
+  private_key = "${file("concourse.key")}"
+
+  provisioner "local-exec" {
+    command = "sleep 10"
```

Comment:
Unfortunately terraform does fail and the pipeline stops. If you restart the job, then it works.
The sleep allows to make it work the first time.